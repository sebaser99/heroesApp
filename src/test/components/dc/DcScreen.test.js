import { shallow } from 'enzyme';
import {DcScreen} from '../../../components/dc/DcScreen';

describe('vamos a probar el <DcScreen />', ()=>{
    test('debe hacer match con el snapshot', ()=>{
        const wrapper = shallow(<DcScreen />)

        expect(wrapper).toMatchSnapshot()
    })
})