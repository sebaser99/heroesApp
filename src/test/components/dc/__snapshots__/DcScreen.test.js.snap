// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`vamos a probar el <DcScreen /> debe hacer match con el snapshot 1`] = `
<div>
  <h1>
    DC Screen
  </h1>
  <hr />
  <HeroList
    publisher="DC Comics"
  />
</div>
`;
